<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Quiz</class>
 <widget class="QDialog" name="Quiz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>120</y>
     <width>1061</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Otazky</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="yesButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>500</y>
     <width>172</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Ano</string>
   </property>
  </widget>
  <widget class="QPushButton" name="noButton">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>510</y>
     <width>172</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Nie</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveAndExitButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>330</y>
     <width>251</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string>Ulozit odpovede a ukoncit kviz</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
